app/src/androidTest/java/org/xwiki/android/sync/activities/EditContactActivityTest.kt

```diff
@@ -0,0 +1,22 @@
+package org.xwiki.android.sync.activities
+
+import org.junit.After
+import org.junit.Before
+
+/**
+ * EditContactActivityTest
+ */
+class EditContactActivityTest {
```

Comment:
Why have you kept this empty test?

Reply:
I've kept it coz, I'll be starting my work on EditContactActivity in coming weeks in which I'll be writing test cases.